SonarQube.MSBuild.Tasks/Resources.resx

```diff
@@ -193,4 +193,19 @@ Error: {1}</value>
   <data name="AnalyzerSettings_NotSpecifiedInConfig" xml:space="preserve">
     <value>Analyzer settings have not been specified in the analysis config file</value>
   </data>
+  <data name="BuildWrapper_AlreadyAttached" xml:space="preserve">
+    <value>BuildWrapper: the build wrapper is already attached to the current process ({0})</value>
+  </data>
+  <data name="BuildWrapper_AttachedSuccessfully" xml:space="preserve">
+    <value>BuildWrapper: the build wrapper attached successfully</value>
+  </data>
+  <data name="BuildWrapper_FailedToAttach" xml:space="preserve">
+    <value>BuildWrapper: failed to attach to the current process</value>
+  </data>
+  <data name="BuildWrapper_RequiredFileMissing" xml:space="preserve">
```

Comment:
Do we expect the user to encounter any of these errors? If so, is there any action he can take to bypass them, e.g. update to the latest version of the C# plugin etc.?